src/components/TreeSpeciesCard.js

```diff
@@ -38,6 +38,7 @@ function TreeSpeciesCard(props) {
             sx={{
               fontFamily: 'Lato',
               fontWeight: 600,
+              width: [240, 'auto'],
```

Comment:
I think a fixed pixel is not so flexible, and responsive for UI, is there any other solution?

Reply:
If this is because of the long text, maybe we can try some property like: word-break